modules/ROOT/pages/7.5-release-notes.adoc

```diff
@@ -91,7 +91,24 @@ For information on the **Markdown** plugin, see xref:markdown.adoc[Markdown].
 
 The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.
 
-**Comments** Premium plugin includes the following fixes.
+**Comments** Premium plugin includes the following additions and fixes.
+
+==== Comments with Mentions
+// EPIC-111
+
+The **Comments** plugin now supports mentions in comments.
+
+Previously, the **Comments** plugin allowed users to add comments to content in the editor. However, users could not mention other users in the comments sidebar, limiting the ability to notify specific users or direct comments to them.
```

Comment:
```suggestion
```
Since this is a new feature, we shouldn't phrase it like a bug fix. The next paragraph will do